tempesta_fw/t/functional/helpers/prepare.py

```diff
@@ -0,0 +1,21 @@
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+from . import remote
+
+def configure_tcp():
+    """ Configuring TCP for faster reuse the same TCP ports.
+    We start a lot of backends and clients in stress tests.
+    And in next stress test we sometimes failing using the
```

Comment:
Better to avoid `we` or `I` if possible.
'We start a lot of backends and clients in stress tests. And in next stress test we sometimes failing using the same ports.' -> 'A lot of sockets are created in tests and bound to specific ports. Release them quicker to reuse the ports in the next test case.'